expvar/docs/counter.memstats.by_size.frees.md

```diff
@@ -0,0 +1,8 @@
+---
+title: memstats.by_size.frees
+brief: Cumulative count of heap objects freed in a class. The class is identified by dimension class and it is as described for metric memstats.by_size.size
```

Comment:
Ideally, brief would be briefer (like just the first sentence fragment) and the description below would be longer. This format will make the table take up much more vertical space when they get ported to the product docs.